src/modules/navigator/mission_feasibility_checker.cpp

```diff
@@ -349,24 +361,23 @@ MissionFeasibilityChecker::check_dist_1wp(dm_item_t dm_current, size_t nMissionI
 		for (unsigned i = 0; i < nMissionItems; i++) {
 			if (dm_read(dm_current, i,
 					&mission_item, sizeof(mission_item_s)) == sizeof(mission_item_s)) {
-				/* Check non navigation item */
-				if (mission_item.nav_cmd == NAV_CMD_DO_SET_SERVO){
-
-					/* check actuator number */
-					if (mission_item.params[0] < 0 || mission_item.params[0] > 5) {
-						mavlink_log_critical(_mavlink_log_pub, "Actuator number %d is out of bounds 0..5", (int)mission_item.params[0]);
-						warning_issued = true;
-						return false;
-					}
-					/* check actuator value */
-					if (mission_item.params[1] < -2000 || mission_item.params[1] > 2000) {
-						mavlink_log_critical(_mavlink_log_pub, "Actuator value %d is out of bounds -2000..2000", (int)mission_item.params[1]);
-						warning_issued = true;
-						return false;
-					}
-				}
-				/* check only items with valid lat/lon */
-				else if (isPositionCommand(mission_item.nav_cmd)) {
+                                /* Check non navigation item */
```

Comment:
Hello Lorenz,
   I have sent the pull request from my new github account "px4user". 
In your comment we have a check of output first and next the check of desired PWM value.
But generally in module mission_feasibility_checker.cpp we have function checkMissionItemValidity where we check all mission items 
  Als we have function check_dist_1wp where some body have initially added check of DO_SET_SERVO mission item. From my point of view it can be excluded. But as is not me who have added this i prefered to keep it.
Regards,
Michail.